core/src/main/codegen/templates/Parser.jj

```diff
@@ -5127,8 +5127,6 @@ SqlIntervalQualifier TimeUnit() : {
 |   <MINUTE> { return new SqlIntervalQualifier(TimeUnit.MINUTE, null, getPos()); }
 |   <HOUR> { return new SqlIntervalQualifier(TimeUnit.HOUR, null, getPos()); }
 |   <DAY> { return new SqlIntervalQualifier(TimeUnit.DAY, null, getPos()); }
-|   <DAYOFWEEK> { return new SqlIntervalQualifier(TimeUnit.DOW, null, getPos()); }
```

Comment:
could you explain why this TOKEN is removed ?

Reply:
DAYOFWEEK is a valid time unit for BigQuery. However here (which I added this in a previous commit), it translates it to DOW which is not a valid time unit for BigQuery. Instead, I wanted to treat it as a timeframe that is an alias to DOW (and DOY, respectively).